openbb_terminal/cryptocurrency/onchain/shroom_view.py

```diff
@@ -18,6 +19,45 @@
 logger = logging.getLogger(__name__)
 
 
+@log_start_end(log=logger)
+@check_api_key(["API_SHROOM_KEY"])
+def display_query(
+    query: str,
+    raw: bool = False,
+    limit: int = 10,
+    export: str = "",
+):
+    """Display query results from shroom
+    [Source: https://sdk.flipsidecrypto.xyz/shroomdk]
+
+    Parameters
+    ----------
+    query : str
+        Query string
+    raw : bool
+        Show raw data
+    limit : int
+        Limit of rows
+    export : str
+        Export dataframe data to csv,json,xlsx file
+    """
+    df = get_query_data(query)
+    if df.empty:
+        console.print("[red]No data found.[/red]")
+    elif not df.empty:
+        if raw:
+            console.print(df)
+        else:
+            print_rich_table(df.head(limit))
```

Comment:
dont think this needs a raw input.  raw differentiates between showing a chart and a table